crates/tessera-backbone/src/server.rs

```diff
@@ -0,0 +1,22 @@
+use axum::Router;
+use tracing::debug;
+
+use crate::config::ApplicationConfig;
+
+pub(super) struct ServerConfig {
+    pub port: u16,
+}
+
+impl From<&ApplicationConfig> for ServerConfig {
+    fn from(value: &ApplicationConfig) -> Self {
+        Self { port: value.port }
+    }
+}
+
+pub(super) async fn run(config: ServerConfig) -> anyhow::Result<()> {
+    let app = Router::new();
```

Comment:
It seems that the logic for initializing the tracing-subscriber instance is missing here

Reply:
subscriber is initilized in the main function.